src/main/webapp/front-end/src/utilities.js

```diff
@@ -1,8 +1,10 @@
+import store from './store.js';
+import { login, loginError, loginSuccess, loginClearInputs } from './actions.js';
 var _ = require('lodash');
 
 export var authenticatedSession = null;
 
-export var authenticationHeader = "Bee-Session-Key"
+export var authenticationHeader = "bee-session-key"
```

Comment:
it's undefined when uppercase even though it shows up as upper case in the Network Tab, I'll investigate further